dvc/command/completion.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+import logging
+import os
+
+import shtab
+
+from dvc.command import choices
+from dvc.command.base import CmdBase, append_doc_link
+
+logger = logging.getLogger(__name__)
+DEFAULT_FILENAME = {"bash": "dvc", "zsh": "_dvc"}
+CHOICE_FUNCTIONS = {
+    "bash": {"DVCFile": "_dvc_compgen_DVCFiles"},
+    "zsh": {"DVCFile": "_files -g '(*.dvc|Dvcfile)'"},
+}
+PREAMBLE = {
+    "bash": """
+# $1=COMP_WORDS[1]
+_dvc_compgen_DVCFiles() {
+  compgen -f -X '!*?.dvc' -- $1  # DVC-files
+  compgen -d -S '/' -- $1  # recurse into subdirs
+}
+""",
+    "zsh": "",
+}
```

Comment:
maybe makes sense to move this to `dvc/command/choices.py` since that's where the `"DVCFile"` choice is defined?

Reply:
@casperdcl Or move the contents of dvc/command/choices.py to dvc/command/completion.py :D I did find it confusing when reviewing.

Reply:
sure - I picked `choices` as a name though to match the `argparse` kwarg it's intended for...

Reply:
moved